### PR TITLE
fix(csh): focus on collaborator avatars

### PR DESCRIPTION
**Pull request by jpan-box**
BEFORE: AutoFocus set to "Invite people..." text area
AFTER: AutoFocus set to "Shared with..." button
Screenshot 2023-04-17 at 1 20 12 PM

### CODE REVIEW

File: src/features/collaborator-avatars/CollaboratorAvatars.js [hunk 1]
```diff
@@ -61,12 +61,13 @@ class CollaboratorAvatars extends Component<Props> {
 
         return (
             <PlainButton
+                aria-hidden={this.isVisible() ? 'false' : 'true'}
+                autoFocus
```
[the remark this answers is not in the document]
Comment (tjuanitas) [reply src/features/collaborator-avatars/CollaboratorAvatars.js:65]: would it work if you updated something here? https://github.com/box/box-ui-elements/blob/master/src/features/unified-share-modal/UnifiedShareModal.js#L164-L170

[the remark this answers is not in the document]
Comment (jpan-box) [reply src/features/collaborator-avatars/CollaboratorAvatars.js:65]: Should we change the logic for USM though? I thought just the CSH element.

[the remark this answers is not in the document]
Comment (tjuanitas) [reply src/features/collaborator-avatars/CollaboratorAvatars.js:65]: USM and CSH are essentially the same thing. CSH just wraps USM so if CSH has an accessibility issue, then the same would apply to USM

**Comment by tjuanitas** [comment]
Closing since we decided that the original issue is not something we want to fix at the moment.

**Pull request closed without merging**